qcodes/tests/test_delegate_parameter.py

```diff
@@ -277,3 +276,21 @@ def test_raw_value_scaling(make_observable_parameter):
     d(val)
     assert d.raw_value == val * 5 + 3
     assert d.raw_value == p()
+
+
+def test_setting_initial_value_delegate_parameter():
+    value = 10
+    p = Parameter('testparam', set_cmd=None, get_cmd=None)
+    d = DelegateParameter('test_delegate_parameter', p,
+                          initial_value=value)
+    assert p.cache._value == value
+    assert d.cache._value == value
+
+
+def test_setting_initial_cache_delegate_parameter():
+    value = 10
+    p = Parameter('testparam', set_cmd=None, get_cmd=None)
+    d = DelegateParameter('test_delegate_parameter', p,
+                          initial_cache_value=value)
+    assert p.cache._value == value
+    assert d.cache._value == value
```

Comment:
Replace with a call to get(get_if_invalid=False) once https://github.com/QCoDeS/Qcodes/pull/2008 lands